sympy/physics/units.py

```diff
@@ -260,6 +260,7 @@ def free_symbols(self):
 avogadro_number = Rational('6.022140857') * 10**23
 avogadro = avogadro_constant = avogadro_number / mol
 boltzmann = Rational('1.38064852') * ten**-23 * J / K
+atomic_mass_constant = Rational('1.660538921') * ten**-27 * kg
```

Comment:
Thanks. I took the value from NIST reference which might be out of date. I'll fix this ASAP.